src/DIRAC/ConfigurationSystem/Client/LocalConfiguration.py

```diff
@@ -386,7 +372,6 @@ def __parseCommandLine(self):
         for opt, val in opts:
             if opt in ("-h", "--help"):
                 self.showHelp()
-                sys.exit(2)
```

Comment:
Why? Running `dirac-something --help` shouldn't ever do anything (though I think it should probably exit with `0`).

Reply:
`showHelp` will do DIRAC.exit()